### PR TITLE
Feature/ Handover

### PR DESCRIPTION
**Pull request by olerichard**
# Handover app 

Implemented Handover Garden from Fusion 

Completes: [AB#160013](https://dev.azure.com/Equinor/Johan%20Castberg%20-%20Portal/_backlogs/backlog/Johan%20Castberg%20-%20Portal%20Team/Features/?workitem=160013)

## Checklist:

-   [ ] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [ ] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/Handover/Garden/CustomViews/HandoverGardenItem.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+const HandoverExpandedTitle = styled.div`
+    display: flex;
+    flex: 1;
+    padding: 0px 8px;
+`;
+
+const Circles = styled.div`
+    display: flex;
+    padding-right: 0px 8px;
+`;
```
Comment (jaysencpp) [inline src/apps/Handover/Garden/CustomViews/HandoverGardenItem.tsx:52]: Move to styles file

File: src/apps/Handover/Garden/components/HandoverSidesheet/HandoverSidesheetTabs/DetailsTab.tsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+import { HandoverDetails, HandoverPackage } from '../../../models';
+
+const StringCell = ({ value }: { value: string }) => <>{value.trim() ? value.trim() : 'N/A'}</>;
+
+const formatDateString = (dateString: string): string => {
+    const date = new Date(dateString);
+    if (date.toString() === 'Invalid Date') return 'N/A';
+    const dateParts = new Intl.DateTimeFormat(undefined).formatToParts(date);
+    return `${dateParts[0].value}/${dateParts[2].value}/${dateParts[4].value}`;
+};
```
Comment (jaysencpp) [inline src/apps/Handover/Garden/components/HandoverSidesheet/HandoverSidesheetTabs/DetailsTab.tsx:13]: utils

File: src/apps/Handover/Garden/hooks/useHandoverDataproxy.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    const getSignatures = useCallback(async () => {
+        setDataFetching(true);
+
+        const packageString = getPackages
+            ? `/${getPackages.selectedPackageId}/${getPackages.packageType}`
+            : '';
+
+        try {
+            const result = await apiClient.fetch(
+                `https://pro-s-dataproxy-fprd.azurewebsites.net/api/contexts/3380fe7d-e5b7-441f-8ce9-a8c3133ee499/handover/${packageString}`
```
Comment (jaysencpp) [inline src/apps/Handover/Garden/hooks/useHandoverDataproxy.ts:40]: change to ci

**Review by Gustav-Eikaas — APPROVED** [review]
LGTM

**Pull request merged**